assertj-android-appcompat-v7/src/main/java/org/assertj/android/appcompat/v7/api/widget/SearchViewAssert.java

```diff
@@ -3,12 +3,12 @@
 
 import android.support.v4.widget.CursorAdapter;
 import android.support.v7.widget.SearchView;
-import org.assertj.android.api.widget.AbstractLinearLayoutAssert;
+import org.assertj.android.api.view.AbstractViewGroupAssert;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Assertions for {@link SearchView} instances. */
-public class SearchViewAssert extends AbstractLinearLayoutAssert<SearchViewAssert, SearchView> {
+public class SearchViewAssert extends AbstractViewGroupAssert<SearchViewAssert, SearchView> {
```

Comment:
This is a potentially breaking change (`SearchView` extends from `LinearLayoutCompat` instead of `LinearLayout` so we can't use the previous base class)